docs/SUMMARY.md

```diff
@@ -46,11 +46,15 @@
 * [Dockerfile containers](usage/dockerfile.md)
 * [Windows support](usage/windows_support.md)
 
+## Continuous Integration
+
+* [CI with shared runners at Gitlab](ci/ci.md#gitlab)
+
 ## Examples
 * [Selenium](https://github.com/testcontainers/testcontainers-java-examples/blob/master/selenium-container/src/test/java/SeleniumContainerTest.java)
 * [Custom Redis container](https://github.com/testcontainers/testcontainers-java-examples/blob/master/redis-backed-cache/src/test/java/RedisBackedCacheTest.java)
 * [Spring Boot testing with Redis](https://github.com/testcontainers/testcontainers-java-examples/tree/master/spring-boot/src/test/java/com/example)
-
+* [Grails app testing RabbitMQ + MySQL (spanish)](https://gitlab.com/jorge-aguilera/pingpong)
```

Comment:
I like it, but we were trying to keep our examples under https://github.com/testcontainers org so that we can control/update them. WDYT if we temporary remove it from this PR and do the example separately?